scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2459,7 +2459,7 @@ void BPF_STRUCT_OPS(layered_stopping, struct task_struct *p, bool runnable)
 	s32 task_lid;
 	u64 runtime;
 
-	if (!(cpuc = lookup_cpu_ctx(-1)) || !(taskc = lookup_task_ctx(p)))
+	if (!(cpuc = lookup_cpu_ctx(-1)) || !(taskc = lookup_task_ctx_may_fail(p)))
```

Comment:
I'm not sure on this one, I'd assume stopping is always called so maybe this isn't required.